contacts.css

```diff
@@ -0,0 +1,195 @@
+.main-container
+{
+    width: 1024px;
+    margin: auto;
+    display: flex;
+    flex-direction: column;
+    gap: 25px;
+}
```

Comment:
To keep the code consistent with the rest of the project, I recommend using `.main-container {` format of styling (not { on the next line). You can automatically apply this by using "Find and replace" in vscode or by using an online CSS formatter

Reply:
Thanks so much for all of your comments. I will fix all of errors that you mention for and  repush the request♡